.github/workflows/build-dumb-pypi.yml

```diff
@@ -0,0 +1,177 @@
+        required: true
+      ansible-base-ref:
+        default: devel
+        description: The GitHub repo ref for pulling ansible-base from
+        required: true
+      ansible-build-data-slug:
+        default: ansible-community/ansible-build-data
+        description: The GitHub repo slug for pulling ansible build data from
+        required: true
+      ansible-build-data-ref:
+        default: main
+        description: The GitHub repo ref for pulling ansible build data from
+        required: true
+  schedule:
+  # https://crontab.guru/#23_*/4_*_*_*
+  - cron: 23 */4 * * *  # At minute 23 past every 4th hour.
+
+jobs:
+  build-dumb-pypi:
+    name: Build ACD and ansible-base nightlies
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Switch to Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Figure out the source repos
+      id: build-settings
+      run: |
+        print(
+            '::set-output name=ansible-base-slug::${{
+                github.event_name == 'workflow_dispatch' &&
+                github.event.inputs.ansible-base-slug ||
+                'ansible/ansible'
+            }}'
+        )
+        print(
+            '::set-output name=ansible-base-ref::${{
+                github.event_name == 'workflow_dispatch' &&
+                github.event.inputs.ansible-base-ref ||
+                'devel'
+            }}'
+        )
+        print(
+            '::set-output name=ansible-build-data-slug::${{
+                github.event_name == 'workflow_dispatch' &&
+                github.event.inputs.ansible-build-data-slug ||
+                'ansible-community/ansible-build-data'
+            }}'
+        )
+        print(
+            '::set-output name=ansible-build-data-ref::${{
+                github.event_name == 'workflow_dispatch' &&
+                github.event.inputs.ansible-build-data-ref ||
+                'main'
+            }}'
+        )
+      shell: python
+    - name: Check out antsibull src from Git
+      uses: actions/checkout@v2
+    - name: >-
+        Check out ${{
+          steps.build-settings.outputs.ansible-build-data-slug
+        }}@${{
+          steps.build-settings.outputs.ansible-build-data-ref
+        }} src from Git
+      uses: actions/checkout@v2
+      with:
+        path: ansible-build-data
+        ref: ${{ steps.build-settings.outputs.ansible-build-data-ref }}
+        repository: ${{ steps.build-settings.outputs.ansible-build-data-slug }}
+    - name: Make 'dist/' dir
+      run: |
+        from pathlib import Path
+        Path('dist').mkdir()
+      shell: python
+    - name: Install packaging-related dists from PyPI
+      run: >-
+        python -m
+        pip install
+        -U --user
+        pip setuptools wheel
+    - name: Install antsibull dist from PyPI
+      run: python -m pip install -U --user antsibull
+    - name: Install antsibull dist from GitHub
```

Comment:
@gundalow @abadger this currently uses the bleeding-edge for antsibull from Git rather than getting it from PyPI. This may be a reason to keep the workflow here + it'll update the index when the antsibull repo itself gets updates...

Reply:
I was thinking that it should be put into the nightly-build repository (keep the artifacts and the tools close together) until this.  I do wonder if the nightly build should use antsibull from git, though, or stick with the antsibull on pypi?

Reply:
After I gave it some thought, I decided to try having it here because the builder is here too. Also, when I was first trying to make it work, PyPI version didn't have all the necessary bits for some reason and I needed something unreleased. I think if we keep it here, it'll also test that antsibull's builder is healthy: it seems reasonable to ensure that it doesn't break with some random change to antsibull.

Of course, we can move it later if it proves to be too annoying.

Reply:
This would also support goals declared in #98 — how can we build a release automation if the tooling does not test this case?

Reply:
I think testing should be in integration and unit testing.  Anything that results in artifacts that the public can access should be considered part of production infrastructure instead.

Reply:
I would agree for the regular tagged releases that are based on a human sign-off but nightlies are supposed to be made out of potentially broken things and I consider them a part of smoke and integration test infrastructure. OTOH if the tool breaks on nightlies, it's not the end of the world but is still a strong signal that something needs to be fixed... To me, it sounds better than finding out that it doesn't work right before making a release.

Reply:
To be clear, as long as we're using a git checkout of the antsibull git repo, I think it is good to host it here.  I'm just not sure if we want to keep doing that down the road.